modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,79 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+====
+
+In this table, replace ``version`` with the version of the product you are using.
+
+[cols="1,1", options="header"]
+.Hardware and Software Requirements
+|===
+| Requirement            | Description
+| openSUSE Leap 15.1     | Clean installation, up-to-date
+| Multi-core CPU         | x86_64
+| 8 GB RAM *) 	         | Sufficient for testing purposes
+| 100 GB Disk Space **)  | Disk space depends on your channel requirements
+|===
+
+*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).
+
+**) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.
+
+
+////
```

Comment:
Should we remove this?

Reply:
@juliogonzalez What do you want to remove?  The storage space requirements or the slashes (`////`)?  The slashes are a block comment start marker.  It makes the following NOTE invisible.  I've no strong opinion on those.